package-lock.json

```diff
@@ -1,9 +1,2325 @@
 {
   "name": "twitch_tools",
   "version": "1.0.0",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
   "requires": true,
+  "packages": {
```

Comment:
is this 3000+ lines change necessary?

Reply:
Hi. Looks like these are dependencies from nodemon.

I ran `npm audit --json` and nodemon seems to add 117 dependencies.
```
{
  "auditReportVersion": 2,
  "vulnerabilities": {},
  "metadata": {
    "vulnerabilities": {
      "info": 0,
      "low": 0,
      "moderate": 0,
      "high": 0,
      "critical": 0,
      "total": 0
    },
    "dependencies": {
      "prod": 114,
      "dev": 117,
      "optional": 1,
      "peer": 0,
      "peerOptional": 0,
      "total": 230
    }
  }
}
```